src/app/core/models/checkout/checkout-step.type.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * available, ordered checkout steps
+ */
+export enum CheckoutStepType {
+  addresses = 1,
```

Comment:
Because these are like constants I would prefer to write them with a starting capital letter as it is in all other occurances, see e.g. here: https://www.typescriptlang.org/docs/handbook/enums.html